site/docs/v1/tech/apis/form-fields/_form-field-type.adoc

```diff
@@ -13,4 +13,9 @@ When using the `date` type, the only supported data format is ISO-8601 or `YYY-M
 +
 When using the `consent` type, the field [field]#consentId# must be included in the request.
 +
-Defaults to `string`, unless `user.birthDate`, then `date`, or if `user.email` then `email`.
\ No newline at end of file
+Defaults can change depending on the type of `key`. The defaults are:
```

Comment:
Defaults are only documented on the request, so when this include is used on a response we don't want to display this part. See examples in the response codes, or other includes for using `ifdef` checks to show or hide sections of content.